native/acter/src/api/push/notification_item.rs

```diff
@@ -157,138 +159,6 @@ impl NotificationRoom {
             .await?
     }
 }
-#[derive(Clone, Debug)]
```

Comment:
this has all moved to the new `Activity` area, which has stayed mostly the same internally though